sdk/keyvault/Azure.ResourceManager.KeyVault/api/Azure.ResourceManager.KeyVault.netstandard2.0.cs

```diff
@@ -284,6 +326,47 @@ public enum AccessPolicyUpdateKind
         Replace = 1,
         Remove = 2,
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ActivationStatus : System.IEquatable<Azure.ResourceManager.KeyVault.Models.ActivationStatus>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ActivationStatus(string value) { throw null; }
+        public static Azure.ResourceManager.KeyVault.Models.ActivationStatus Active { get { throw null; } }
+        public static Azure.ResourceManager.KeyVault.Models.ActivationStatus Failed { get { throw null; } }
+        public static Azure.ResourceManager.KeyVault.Models.ActivationStatus NotActivated { get { throw null; } }
+        public static Azure.ResourceManager.KeyVault.Models.ActivationStatus Unknown { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.KeyVault.Models.ActivationStatus other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.KeyVault.Models.ActivationStatus left, Azure.ResourceManager.KeyVault.Models.ActivationStatus right) { throw null; }
+        public static implicit operator Azure.ResourceManager.KeyVault.Models.ActivationStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.KeyVault.Models.ActivationStatus left, Azure.ResourceManager.KeyVault.Models.ActivationStatus right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class Attributes
```

Comment:
this name is too generic, please find the type or property that are using this type and prepend a proper prefix to this type

Reply:
rename to `SecretBaseAttributes`